core/services/ocr2/plugins/ocr2keeper/evm21/streams_lookup_test.go

```diff
@@ -208,7 +244,7 @@ func TestEvmRegistry_StreamsLookup(t *testing.T) {
 					}).Once()
 			}
 
-			if len(tt.blobs) > 0 {
+			if len(tt.blobs) > 0 && !tt.v3 {
```

Comment:
Could this if and the following if become:

```
if len(tt.blobs) > 0 {
   if tt.v3 {

   } else {

   }
}
```

Instead of

```
if len(tt.blobs) > 0 && !tt.v3 {

}
if len(tt.blobs) > 0 && tt.v3 {

}
```